README.MD

```diff
@@ -38,10 +38,10 @@ Presidio _(Origin from Latin praesidium ‘protection, garrison’)_ helps to en
 
 ## Installing Presidio
 
-1. [Using pip](https://microsoft.github.io/presidio/installation/#using-pip)
-2. [Using Docker](https://microsoft.github.io/presidio/installation/#using-docker)
-3. [From source](https://microsoft.github.io/presidio/installation/#install-from-source)
-4. [Supported Python Versions](https://microsoft.github.io/presidio/installation/#supported-python-versions)
+1. [Supported Python Versions](https://microsoft.github.io/presidio/installation/#supported-python-versions)
```

Comment:
where is it suppose to refer to? the link just seems to go to installation.

Reply:
once we deploy the latest docs to github pages it will refer to the supported python versions section